recipe/build.sh

```diff
@@ -1,11 +0,0 @@
-#!/bin/bash
-
-if [[ $(uname) == Darwin ]]; then
-  export LDFLAGS="-headerpad_max_install_names $LDFLAGS"
-fi
-
-$PYTHON -m pip install --no-deps --ignore-installed . \
-                       --global-option=build_ext \
-                       --global-option="-I$PREFIX/include" \
-                       --global-option="-L$PREFIX/lib" \
-                       --global-option="-lgdal"
```

Comment:
I think this change is fine. For Linux/MacOS, fiona's setup.py should find those paths automatically through `gdal-config`